### PR TITLE
[RocksDB] [Draft: waiting for upstream] Initial integration

### PR DESCRIPTION
**Pull request by DavidKorczynski**

### CODE REVIEW

File: projects/rocksdb/build.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
+# Copyright 2020 Luca Boccassi <bluca@debian.org>
```
Comment (jonathanmetzman) [inline projects/rocksdb/build.sh:3]: Can you get rid of this line please?

**Review by jonathanmetzman — APPROVED** [review]
We approve of adding this project.
Just let us know when/if it is ready to land.

**Comment by mhlakhani** [comment]
Thanks for putting this fuzzer up for review! can you please add `oss-fuzz@fb.com` to the vendor_ccs (similar to https://github.com/google/oss-fuzz/blob/master/projects/proxygen/project.yaml ?) and `mhl@fb.com` to the `auto_ccs` as well?

**Comment by DavidKorczynski** [comment]
@jonathanmetzman This one is ready to land now. 

Tagging @cheng-chang for convenience

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**